modules/php/States/SelectCardTrait.php

```diff
@@ -49,26 +51,19 @@ public function argSelect()
   public function select($ids)
   {
     $args = Log::getLastAction('selection');
```

Comment:
Remove that line and use stackContext instead, the player id is right into the pId field !

Reply:
Got it, changed